arrow-fx-reactor/src/main/kotlin/arrow/fx/reactor/extensions/fluxk.kt

```diff
@@ -196,79 +197,107 @@ interface FluxKConcurrent : Concurrent<ForFluxK>, FluxKAsync {
 
   override fun <A, B> CoroutineContext.racePair(fa: FluxKOf<A>, fb: FluxKOf<B>): FluxK<RacePair<ForFluxK, A, B>> =
     asScheduler().let { scheduler ->
+      val active = AtomicBooleanW(true)
+      val forward = Disposables.composite()
+
       Flux.create<RacePair<ForFluxK, A, B>> { emitter ->
         val sa = ReplayProcessor.create<A>()
         val sb = ReplayProcessor.create<B>()
 
+        emitter.onCancel(forward)
+
         val dda = fa.value()
           .subscribeOn(scheduler)
           .subscribe(sa::onNext, sa::onError)
+          .also { forward.add(it) }
```

Comment:
NIT: `also(forward::add)`

Reply:
That bailed with a compile error, `TypeVariable<(U)>`. Not sure what is going wrong there.